src/blueapi/service/handler.py

```diff
@@ -92,18 +94,9 @@ def setup_handler(
     plan_wrappers = []
 
     if config:
-        visit_service_client: VisitServiceClientBase
-        if config.env.data_writing.visit_service_url is not None:
-            visit_service_client = VisitServiceClient(
-                config.env.data_writing.visit_service_url
-            )
-        else:
-            visit_service_client = LocalVisitServiceClient()
-
-        provider = VisitDirectoryProvider(
-            data_group_name=config.env.data_writing.group_name,
-            data_directory=config.env.data_writing.visit_directory,
-            client=visit_service_client,
+        provider = StaticDirectoryProvider(
+            filename_prefix="blueapi",
```

Comment:
This could actually be `f"{config.env.data_writing.group_name}-blueapi"`, which would mean the files are called `<BEAMLINE>-blueapi-<UUID>.h5`, possibly slightly easier to interpret.

Reply:
Does it mean that?

When I've been testing this locally (i.e. with static directory provider) I don't recall the filenames of e.g. the hdf writer having uuid's in them. I think this will just produce a `<BEAMLINE>-blueapi.h5` file, which is fine for now.